src/devices/bus/s100/vectordualmode.cpp

```diff
@@ -228,26 +227,23 @@ bool s100_vector_dualmode_device::get_next_bit(attotime &tm, const attotime &lim
 	return true;
 }
 
-void s100_vector_dualmode_device::floppy_sector_hole_cb(floppy_image_device *floppy, int state)
```

Comment:
@galibert Some 8" floppy drives separated the sector and index hole signals. The 50-pin cable had a separate sector line (pin 24) so we may want to consider having a separate CB if we want to support 8" hard-sectored floppy.